src/forms/renderers/Overrides/material/controls/MaterialDateControl.tsx

```diff
@@ -103,14 +101,15 @@ export const Custom_MaterialDateControl = (props: ControlProps) => {
                             }
                         )}
                         disabled={!enabled}
-                        {...bindTrigger(popupState)}
+                        ref={buttonRef}
+                        {...bindFocus(state)}
```

Comment:
We now use focus so that when a user is tabbing through the UI the popover closes correctly.